agent/ecs_client/model/api/api-2.json

```diff
@@ -1475,6 +1507,13 @@
         "taskDefinition":{"shape":"TaskDefinition"}
       }
     },
+    "RepositoryCredentials":{
+      "type":"structure",
+      "required":["credentialsParameter"],
+      "members":{
+        "credentialsParameter":{"shape":"String"}
```

Comment:
Does this allow for more than one credential set? And is that something we should consider?